src/puptoo/app.py

```diff
@@ -232,6 +232,9 @@ def handle_message(msg, service):
             facts["display_name"] = msg["metadata"]["display_name"]
         if msg["metadata"].get("ansible_host"):
             facts["ansible_host"] = msg["metadata"]["ansible_host"]
+
+        # delete yum_updates from system_profile as it is already present under custom_metadata 
+        del facts["system_profile"]["yum_updates"]
```

Comment:
@r14chandra - could you please try by moving this line under above `if block` [here](https://github.com/RedHatInsights/insights-puptoo/pull/222/files#diff-fa9bb8d3ec0a516b367a439fc50fa2e681b49b61101455c393da82788742fbbfR220)?

Reply:
Note that `yum_updates` data is not present under test classic archive file which is causing tests failures.